Info.plist

```diff
@@ -93,6 +93,26 @@
 			<key>CFBundleTypeRole</key>
 			<string>Viewer</string>
 		</dict>
+		<dict>
+			<key>CFBundleTypeExtensions</key>
+			<array>
+				<string>x3d</string>
+			</array>
+			<key>CFBundleTypeName</key>
+			<string>X3D file</string>
+			<key>CFBundleTypeRole</key>
+			<string>Viewer</string>
```

Comment:
We cannot import X3D files, so this entire block should be removed. Same for OBJ.